cpp/bench/ann/src/raft/raft_ivf_pq_wrapper.h

```diff
@@ -78,7 +78,7 @@ class RaftIvfPQ : public ANN<T> {
   {
     AlgoProperty property;
     property.dataset_memory_type = MemoryType::Host;
-    property.query_memory_type   = MemoryType::Device;
+    property.query_memory_type   = MemoryType::Host;
```

Comment:
I am surprised that this does not break the code. RAFT `ivf_pq::search` API expects queries on device.

Reply:
It does break I think. These changes and the debug (print) statements were all part of some experiments that I was doing. Updating all of those now.